std/bigint.d

```diff
@@ -53,6 +53,13 @@ public:
     /// BUG: Should throw a IllegalArgumentException/ConvError if invalid character found
     this(T : const(char)[] )(T s) pure
     {
+        if (s.length == 0)
+        {
+            data = 0UL;
+            sign = false;
+            assert(isZero());
+            return;
```

Comment:
Decided to just initialize a zero BigInt instead of throwing or asserting, as I didn't want to introduce possible breakage in `nothrow` code, and the original bug report was complaining about array overrun in `-release` (no-assert) mode.